dialect_cockroach.go

```diff
@@ -259,7 +258,6 @@ func (p *cockroach) TruncateAll(tx *Connection) error {
 		}
 	}
 	return nil
-	// return tx3.RawQuery(fmt.Sprintf("truncate %s cascade;", strings.Join(tableNames, ", "))).Exec()
```

Comment:
I just leave it to remember it as TODO item. Can you please add TODO comments here?

Reply:
Sure.